docs/contributing.md

```diff
@@ -126,16 +126,47 @@ Customizing the lakeFS docs site should follow the following guidelines: [Just T
 * You can explain things better by including examples. Show, not tell. Use illustrations, images, gifs, code snippets, etc.
 * Establish a visual hierarchy to help people quickly find the information they need. Use text formatting to create levels of title and subtitle (such as h1 to h6 headings in HTML).
 
-### Test your changes localy
-To render the documentation locally and preview changes, use the following command and [browse the documentation locally](http://localhost:4000):
+### Test your changes locally
 
-```sh
-cd docs
-docker run --rm -p 4000:4000 --volume="$PWD:/srv/jekyll:Z" -it jekyll/jekyll:3.8 jekyll serve
-```
+To render the documentation locally and preview changes you can run the Jeykll server under Docker. 
+
+1. Launch the Docker container:
+
+   ```sh
+   docker run --rm \
+              --publish 4000:4000 --publish 35729:35729 \
+              --volume="$PWD/docs:/srv/jekyll:Z" \
+              --volume="$PWD/docs/.jekyll-bundle-cache:/usr/local/bundle:Z" \
+              --interactive --tty \
+              jekyll/jekyll:3.8 \
+              jekyll serve --livereload
+   ```
 
+2. The first time you run the container it will need to download dependencies and will take several minutes to be ready. 
+
+   Once you see the following output, the docs server is ready to [open in your web browser](http://localhost:4000): 
+
+   ```
+   Server running... press ctrl-c to stop.
+   ```
+
+3. When you make a change to a page's source the server will automatically rebuild the page which will be shown in the server log by this entry:
+
+   ```
+   Regenerating: 1 file(s) changed at 2023-01-26 08:34:47
+                  contributing.md
+   Remote Theme: Using theme pmarsceill/just-the-docs
+   ```
+
+   This can take a short while—you'll see something like this in the server's output when it's done. 
+   
+   ```
+   ...done in 34.714073461 seconds.
+   ```
 
+   Your page will automatically reload to show the changes.
 
+_If you are doing lots of work on the docs you may want to leave the Docker container in place (so that you don't have to wait for the dependencies to load each time you re-create it). To do this remove the `--rm` from the `docker run` command._
```

Comment:
Do you mean to recommend keep running the service in the background? Removing the '--rm' flag and run as a daemon using `-d`? We can also recommend adding a name like `--name docs`, so in case we like to stop it, we can use the name and not try to lookup for the right container.

Reply:
thanks @nopcoder, added.